### PR TITLE
SMP-1999: increase replica count for services in prod config

### PR DESCRIPTION
**Pull request by kapilgarg1996**

### CODE REVIEW

**Review by arya-harness — APPROVED** [review]

File: src/harness/override-prod.yaml [hunk 1]
```diff
@@ -645,7 +660,7 @@ platform:
   scm-service:
     autoscaling:
       enabled: false
-    replicaCount: 1
+    replicaCount: 2
```
Comment (avaakash) [inline src/harness/override-prod.yaml:663]: With auto-scaling enabled, will the replica count go below 2 or act as the minReplica count?

**Review by avaakash — APPROVED** [review]

File: src/harness/override-prod.yaml [hunk 2]
```diff
@@ -496,7 +511,7 @@ platform:
   delegate-proxy:
```
[the remark this answers is not in the document]
Comment (kapilgarg1996) [reply src/harness/override-prod.yaml:511]: yes but we have pdb for it and it can block maintenance

File: src/harness/override-prod.yaml (hunk 1 above)
Comment (kapilgarg1996) [reply src/harness/override-prod.yaml:663]: if autoscaling is enabled then customer will set minReplicas accordingly

**Comment by akshay-khole** [comment]
@kapilgarg1996  could you merge this and close ticket https://harness.atlassian.net/browse/SMP-1999

**Pull request merged**